README.md

```diff
@@ -1,5 +1,14 @@
+[![Gluon](.github/assets/gluon_logo.svg)](https://gluonhq.com)
+
 # Gluon Scene Builder #
 
+[![Build](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml/badge.svg)](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml)
+[![License](https://img.shields.io/badge/license-BSD-green)](./LICENSE)
+
+Scene Builder Kit: 
```

Comment:
Can we remove this text? I feel its given that these  badges are for kit.

If we still want to explicitly mention it, I would make it as a header and put the badges below it.

Reply:
done